### PR TITLE
Add EPUB documentation builder

### PR DESCRIPTION
**Pull request by dmitryilyin**
## Add EPUB format builder configuration

While PDF is awesome is many cases it does not work really well on the small screens
of mobile devices. Let's add EPUB format too.

Also, could you please add EPUB files to https://www.sylabs.io/docs/
and please rename PDF files to include version numbers i.e. user-guide-3.0.pdf

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -273,3 +273,46 @@
...
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #texinfo_no_detailmenu = False
+
+# -- Options for Epub output -------------------------------------------------
+
+# Bibliographic Dublin Core info.
+
+# The title of the document. It defaults to the html_title option
+# but can be set independently for epub creation.
+epub_title = 'Singularity User Docs ' + version
```
Comment (jmstover) [inline conf.py]: Can you change this to:

```
epub_title = 'Singularity User Documentation ' + version
```

I'm also somewhat thinking maybe:
```
'Singularity ' + version + ' User Documentation'
```
But, that's a semantics thing on how it gets viewed. The version is the Singularity version, not a documentation versioning... so...  @devprincess @GodloveD

**Review by jmstover — DISMISSED** [review]

File: conf.py (hunk 1 above)
Comment (dmitryilyin) [reply conf.py]: Done

**Review by jmstover — APPROVED** [review]
LGTM... @GodloveD or @devprincess do with it as you will.

**Review by GodloveD — APPROVED** [review]
Also lgtm.  @devprincess

**Comment by GodloveD** [comment]
Thanks very much for these contributions @dmitryilyin !

**Comment by jchavezlavalle** [comment]
lgtm as well, thanks @dmitryilyin !

**Pull request merged**